File: 은지/LV1/이상한 문자 만들기.js
function solution(s) {
  var answer = '';
  var i = 0
  for (let x of s) {
    if (x === " ") {
      answer += x
      i = 0
    }
    else {
      if (i % 2 === 0) {
        answer += x.toUpperCase()
      } else {
        answer += x.toLowerCase()
      }
      i += 1
    }
  }
  return answer;
}

File: 은지/LV1/이상한 문자 만들기.py
def solution(s):
    answer = ''
    i = 0
    for x in s:
        # 띄어쓰기 => i = 0
        if x == " ":
            answer += x
            i = 0
        # 문자
        else:
            if i % 2 == 0:
                answer += x.upper()
            else:
                answer += x.lower()
            i += 1
    return answer